### PR TITLE
Docs on UMB message formats and processors.

### PR DESCRIPTION
**Pull request by mikebonnet**
Generate docs from the test cases. Blatantly ripped off from fedmsg_meta_fedora_infrastructure. :smile:

### CODE REVIEW

**Comment by mikebonnet** [comment]
@ralphbean Please take a look when you have a minute.

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#
+# ToscaWidgets 2 documentation build configuration file, created by
```
Comment (ralphbean) [inline doc/conf.py]: Lol.  ToscaWidgets.  I will never escape you!

File: doc/start.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Getting started with fedmsg_meta_umb
+======================================================
```
Comment (ralphbean) [inline doc/start.rst]: Underline needs to match the text, I think.

File: doc/conf.py (hunk 1 above)
Comment (danc86) [reply doc/conf.py]: Haha. Well, at least it's not the original TurboGears widgets like we have in Beaker :-)

**Pull request merged**